### PR TITLE
Add assignment 4 and lecture 5 slides

### PR DESCRIPTION
**Pull request by ptwu**

### CODE REVIEW

File: versioned_docs/version-2021sp/lecture5.md [hunk 1]
```diff
@@ -3,4 +3,8 @@ id: lecture5
 title: Lecture 5
 ---
 
-Coming soon!
+[Lecture Video](https://drive.google.com/file/d/1rC5MTHQ3vU0vELZyzcUHptZq583VsUAT/view?usp=sharing)
+
+[Lecture Slides](https://docs.google.com/presentation/d/1VseDHeXrYrP2NdsBJTA72mSTlLNUeXeFYkIFUzLuJ5Q/edit?usp=sharing)
```
Comment (Enochen) [inline versioned_docs/version-2021sp/lecture5.md:8]: These are lecture 4 slides/video

**Review by Enochen — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2021sp/lecture5.md (hunk 1 above)
Comment (ptwu) [reply versioned_docs/version-2021sp/lecture5.md:8]: misclick

Comment (ptwu) [reply versioned_docs/version-2021sp/lecture5.md:8]: good catch

**Review by Enochen — APPROVED** [review]
looks good

**Pull request merged**